### PR TITLE
Created OpenSource docs & folders for content

### PR DESCRIPTION
**Pull request by DanM3rcurius**
Created OS docs to be compliant with OS standards, as per #28

### CODE REVIEW

**Comment by DanM3rcurius** [comment]
changed license to [Creative Commons 4.0 Non Commercial Share Alike](https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode)

**Comment by DanM3rcurius** [comment]
@stephen-rowan here as well please if you find the time 🙏🏿

File: Newsletters/SecretDecks to Occultish.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
 ## Intro
 Calling all creators! SecretDecks® looks to you if you’ve ever reinterpreted the Tarot and made your own deck... or perhaps you feel inspired to do so? 
-Are you an artist? Build a new global fanbase, make a living off your digital art without any middlemen, as well as own & manage your copyrights!
-What if you're not an aritst? Join us anyway and find out how you can help us to help each other and what you can get in return! 
+
+### Are you an artist? Do you like Tarot?
+Build a new global fanbase, make a living off your art without any middlemen and through the medium of your choice, as well as own & manage your copyrights!
```
Comment (stephen-rowan) [inline Newsletters/SecretDecks to Occultish.md:5]: I would rewrite as 

"Build a new global fanbase, make a living off your art without any middlemen through the medium of your choice and manage your copyrights !"

File: Newsletters/SecretDecks to Occultish.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+
+### What if you're not an aritst? 
+Join us anyway and find out how you can help us to help each other and what you can get in return!
```
Comment (stephen-rowan) [inline Newsletters/SecretDecks to Occultish.md:8]: Suggested rewrite

"Join us anyway. Help us to help each other.

File: Newsletters/SecretDecks to Occultish.md [hunk 3]
```diff
@@ -1,9 +1,14 @@
 ## Intro
 Calling all creators! SecretDecks® looks to you if you’ve ever reinterpreted the Tarot and made your own deck... or perhaps you feel inspired to do so? 
-Are you an artist? Build a new global fanbase, make a living off your digital art without any middlemen, as well as own & manage your copyrights!
-What if you're not an aritst? Join us anyway and find out how you can help us to help each other and what you can get in return! 
+
+### Are you an artist? Do you like Tarot?
+Build a new global fanbase, make a living off your art without any middlemen and through the medium of your choice, as well as own & manage your copyrights!
+
+### What if you're not an aritst?
```
Comment (stephen-rowan) [inline Newsletters/SecretDecks to Occultish.md:7]: ### What if you're not an artist ?

File: README.md [hunk 4]
```diff
@@ -6,9 +6,20 @@
 
 
 # Contentwriting
-Content for the SecretDecks® brand
+Content for the Autonomous Creator Organisation (ACO) - the body - and SecretDecks (SD) - the soul.
```
Comment (stephen-rowan) [inline README.md:9]: Suggested rewrite

"Content for the **body**  - the Autonomous Creator Organisation (ACO) and for the **soul** - SecretDecks (SD).

**Review by stephen-rowan — CHANGES_REQUESTED** [review]
Hi @DanM3rcurius - made some suggestions before merging.

File: Newsletters/SecretDecks to Occultish.md (hunk 1 above)
Comment (DanM3rcurius) [reply Newsletters/SecretDecks to Occultish.md:5]: very agreeable, shorter and more to the point.

File: Newsletters/SecretDecks to Occultish.md (hunk 2 above)
Comment (DanM3rcurius) [reply Newsletters/SecretDecks to Occultish.md:8]: also here, more precise and catchy wording, thanks Stephen 🙏🏿

File: Newsletters/SecretDecks to Occultish.md (hunk 3 above)
Comment (DanM3rcurius) [reply Newsletters/SecretDecks to Occultish.md:7]: What am I missing here? It looks like you changed nothing?

**Comment by DanM3rcurius** [comment]
for workflow documentation and testing sake: How do you propose I edit in the reviewed changes? I tried adding directly from the PR, but can only add comments.
I now edited the file in my personal repo and am about to create a new PR. but it feels off... I then have two open PRs of which I can only close one. 🤷🏿‍♂️ 
@stephen-rowan thoughts?

File: Newsletters/SecretDecks to Occultish.md (hunk 3 above)
Comment (evolayak) [reply Newsletters/SecretDecks to Occultish.md:7]: think it was just a typo correction for "artist"

Comment (stephen-rowan) [reply Newsletters/SecretDecks to Occultish.md:7]: Yes - just "artist" spelling @DanM3rcurius

**Comment by DanM3rcurius** [comment]
*peeling off facemask that was covering eyes* Oh now I see. Thanks guys 😅

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Review by DanM3rcurius — COMMENTED** [review]
all review requests integrated 🙏🏿

**Comment by DanM3rcurius** [comment]
this PR closes #28

**Pull request merged**